felix/build_fv.sh

```diff
@@ -0,0 +1,20 @@
+# Script Name: build_and_test.sh
```

Comment:
nit: shouldn't this be the same as the filename?

TBH I feel conflicted about adding this script, it doesn't seem add much value, couldn't the commands in it just be in the debugging guide in the README?

Reply:
(just saw Shaun's comment about `make fv-prereqs`, that's much better IMHO )